examples/FirebaseCloudMessaging_Send_ESP8266/FirebaseCloudMessaging_Send_ESP8266.ino

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// FirebaseCloudMessaging_Send_ESP8266 is a sample that shows sending
+// messages to firebase.
+
+#include <ESP8266WiFi.h>
+
+#include <FirebaseCloudMessaging.h>
```

Comment:
We probably should. For this level I Think we should maintain separate objects. For the simple arduino level do we just want one object for discoverability?

(let's discuss I am going to merge this but we can do a followup PR)